src/components/utils/i18n.js

```diff
@@ -0,0 +1,32 @@
+import Polyglot from 'node-polyglot'
+
+import { locales, mobileLocales } from '../../../locales'
+import { isDesktop } from './'
+
+const defaultLocale = 'en'
+
+const unsupportedLocale = () => {
+  console.warn('Locale not supported')
+  return defaultLocale
+}
+
+const setLocale = (language) => {
+  if (!language || !language.locale) return defaultLocale
+  return locales[language.locale] ? language.locale : unsupportedLocale()
+}
+
+const setTestLocale = (phrases) => {
+  // TODO: this is a temporary implementation to work around an error while loading built-in `fs` module
+  // It allows the tests engine to fetch the translations
+  // This data should be stored into a file within the test folder.
+  window.testLocale = phrases
+}
+
+export const setI18n = (language) => {
+  const locale = setLocale(language)
+  const phrases = locales[locale]
+  const polyglot = new Polyglot({locale, phrases, onMissingKey: () => null})
```

Comment:
So if a key is missing we don't show anything? Is this better than defaulting to English?

Reply:
No, because we have some logic that is based on the presence of a string. Maybe this can be refactored later on but changing the logic within the SDK is outside of the scope of this ticket.